### PR TITLE
fix(TBD-10013): correct HdInsight dependencies to fix unit tests

### PR DESCRIPTION
**Pull request by sponomarova**
**Please check if the PR fulfills these requirements**

- [ ] The commit(s) message(s) follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format) ?
- [ ] Unit tests for the Java changes have been added (for bug fixes / features) ?
- [ ] TUJ for the JavaJet changes have been added (for bug fixes / features) ?
- [ ] Docs have been added / updated (for bug fixes / features) ?
- [ ] The new code does not introduce new technical issues

**What is the current behavior?** (You can also link to an open issue here)

**What is the new behavior?**

**BREAKING CHANGE**

If this PR contains a breaking change, please describe the impact and migration
path for existing applications.
If not please remove this section.

**Other information**:

### CODE REVIEW

File: main/plugins/org.talend.hadoop.distribution.hdinsight340/plugin.xml [hunk 1]
```diff
@@ -290,7 +290,7 @@
             <library id="datanucleus-core-3.2.10.jar"/>
             <library id="datanucleus-rdbms-3.2.9.jar"/>
         </libraryNeededGroup>
-        
+        `
```
Comment (CarlosTalend) [inline main/plugins/org.talend.hadoop.distribution.hdinsight340/plugin.xml]: to be removed?

File: main/plugins/org.talend.hadoop.distribution.hdinsight360/plugin.xml [hunk 2]
```diff
@@ -6,8 +6,8 @@
          <!-- hd insight common libraries -->
         <libraryNeeded
             context=""
-            id="javax.ws.rs-api-2.1.jar"
-            name="javax.ws.rs-api-2.1.jar"
+            id="javax.ws.rs-api-2.1-6.0.0.jar"
+            name="javax.ws.rs-api-2.1-6.0.0.jar"
```
Comment (nrousseau) [inline main/plugins/org.talend.hadoop.distribution.hdinsight360/plugin.xml]: this part was correct before.

mvn:org.talend.libraries/javax.ws.rs-api-2.1/6.0.0

=> for all org.talend.libraries, verrsions are stripped.
Means jar generated will be finally: javax.ws.rs-api-2.1.jar

This is not the good place to fix here

Comment (sponomarova) [reply main/plugins/org.talend.hadoop.distribution.hdinsight360/plugin.xml]: updated

**Comment by nrousseau** [comment]
note: approved a bit in advanced to avoid to block the team here, just missing only the change of reference of mvnuri in the components tHCatalog

**Pull request merged**